compiler+runtime/src/cpp/jank/init.cpp

```diff
@@ -0,0 +1,69 @@
+#include <llvm-c/Target.h>
+#include <llvm/Support/CommandLine.h>
+#include <llvm/Support/ManagedStatic.h>
+#include <llvm/Support/TargetSelect.h>
+
+#include <clojure/core_native.hpp>
+#include <clojure/string_native.hpp>
+
+#include <jank/compiler_native.hpp>
+#include <jank/perf_native.hpp>
+#include <jank/init.hpp>
```

Comment:
Including this doesn't get us anything.

Reply:
Without it the linter recommends we make the `init` function static

Reply:
Ah, silly linter. Ok, we can keep it then.